touch_of_cv.py

```diff
@@ -0,0 +1,30 @@
+
+## Library imports
+import numpy as np
+import matplotlib.pyplot as plt
+from PIL import Image
+import os
+from util.math import conv
+from util.image_tools import extract_images
+
+# 0-degree sobel
+sobel = np.array([[-1, 0, 1],[-2, 0, 2],[-1, 0, 1]])
+
+edge_img_paths = extract_images()
+edge_imgs = []
+for path in edge_img_paths:
+    img = Image.open(path).convert('L')
+    np_img = np.asarray(img)
+    edge_img = conv(sobel, np_img)
+    edge_img = np.pad(edge_img, (1, 1), 'constant', constant_values=(0,0))
+    new_img = Image.fromarray(edge_img)
+    edge_imgs.append(new_img)
+
+output_path = "./output"
+os.mkdir(output_path)
```

Comment:
This may error out without this change:
```suggestion
if not os.isdir(output_path):
    os.mkdir(output_path)
```